sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/ImmunizationService.java

```diff
@@ -0,0 +1,55 @@
+package org.sormas.e2etests.services;
+
+import com.github.javafaker.Faker;
+import com.google.inject.Inject;
+import java.time.LocalDate;
+import java.util.UUID;
+import org.sormas.e2etests.enums.CommunityValues;
+import org.sormas.e2etests.enums.DistrictsValues;
+import org.sormas.e2etests.enums.RegionsValues;
+import org.sormas.e2etests.enums.immunizations.ImmunizationManagementStatusValues;
+import org.sormas.e2etests.enums.immunizations.StatusValues;
+import org.sormas.e2etests.pojo.web.Immunization;
+
+public class ImmunizationService {
```

Comment:
I've noticed some problems with the current approach and started refactoring POJO creation.
Faker library generates a random email, which is not linked somehow to user's name, so in case of failure, or debug, is hard to compare POJOs fields (last name, first name, email, linked contacts), so I started refactoring the email to be generated based on user  first+last names @entity name.
Please take a look into this PR and add the same:
https://github.com/hzi-braunschweig/SORMAS-Project/pull/7573